File: 1108-defanging-an-ip-address/1108-defanging-an-ip-address.java
class Solution {
    public String defangIPaddr(String address) {
        String str = "";
        for(int i = 0; i < address.length(); i++){
            if(address.charAt(i) == '.')
                str += "[.]";
            else
                str += address.charAt(i);
        }
        return str;
    }
}

File: 1108-defanging-an-ip-address/1108-defanging-an-ip-address.swift
class Solution {
    func defangIPaddr(_ address: String) -> String {
        var str = String()
        
        for c in address {
            if c == "." {
                str.append("[")
                str.append(".")
                str.append("]")
            } else {
                str.append(c)
            }
        }
        return str
    }
}